app/services/account_reset/delete_account.rb

```diff
@@ -24,8 +24,12 @@ def call
 
     attr_reader :success, :account_age, :mfa_method_counts
 
+    # @return [Integer, nil] number of days since the account was confirmed (rounded) or nil if
+    # the account was not confirmed
     def track_account_age
-      @account_age = ((Time.zone.now - user.confirmed_at) / 1.day).round
+      @account_age = if user.confirmed_at
+                       (Time.zone.now - user.confirmed_at).seconds.in_days.round
+                     end
```

Comment:
Any stylistic preference?

```suggestion
      return if !user.confirmed_at
      @account_age = (Time.zone.now - user.confirmed_at).seconds.in_days.round
```

Reply:
Yes, anything to unindent. Love it.